tasks/main.yml

```diff
@@ -177,6 +177,13 @@
     group: "{{ tomcat_user_group }}"
     mode: 0640
 
+- name: Remove ROOT Webapp Context
+  file:
+    state: absent
+    path: "{{ tomcat_instance_path }}/webapps/ROOT/"
+  when:
+    - not tomcat_root_context
```

Comment:
The only concern is idempotency. 

For example with this conditional, if you were later to place a ROOT context using the `install_webapps_path` then the playbook would remove this context on each new run. 

Which would be less than ideal. For example, if only for first install it could be added with the clause
`not __tomcat_exists.stat.exists`